src/sdk/model/vm/guest-customization/guest-customization.ts

```diff
@@ -0,0 +1,104 @@
+import { GuestCustomizationJson } from './__json__';
+
+/**
+ * VM Guest Customization
+ */
+export class GuestCustomization {
+  protected _json: GuestCustomizationJson;
+
+  constructor(guestCustomization: GuestCustomization);
+  constructor(guestCustomizationJson: GuestCustomizationJson);
+  constructor(firstParam: GuestCustomization | GuestCustomizationJson) {
+    if (firstParam instanceof GuestCustomization) {
+      // Copy constructor
+      this._json = (firstParam as GuestCustomization).json;
+    } else {
+      // Json or empty constructor
+      this._json = (firstParam || {}) as GuestCustomizationJson;
+    }
+  }
+
+  get enabled(): boolean {
```

Comment:
Maybe we can add some doc for those getters ?

Reply:
@fradinni can we add docs on this one?